packages/xstate-react/src/fsm.ts

```diff
@@ -1,35 +1,70 @@
-import { useState, useRef, useEffect } from 'react';
+import { useState, useEffect } from 'react';
 import { StateMachine, EventObject, interpret } from '@xstate/fsm';
 import { AnyEventObject } from 'xstate';
 
+// TS enums don't tree-shake well
+const NotStarted = 'not started';
+const Running = 'running';
+const Stopped = 'stopped';
+
+type InterpreterStatus = typeof NotStarted | typeof Running | typeof Stopped;
+
+const interpretDev: typeof interpret = machine => {
```

Comment:
Is `interpretDev` just for the console error? If so, should this be part of `@xstate/fsm` instead?

Reply:
It's for status tracking and that console.error. This, of course, can be included in `@xstate/fsm` but was not sure if we want to add size to it. Plus in here we can actually give a slightly better error message to the user (for this particular scenario).

Reply:
If we make status a normal enum (`0`, `1`, `2`) it should be fine to include.

Reply:
Ok, gonna split this out from this PR and create a new one for `@xstate/fsm` - after it gets merged in we will be able to iterate on this one.

Reply:
I've rebased this PR on top of the master - so I have already access to `service.status` directly, without tracking it in here. 

Question is - do we want to move this console.error to `@xstate/fsm` itself, convert it to an actual thrown error, catch it here in `interpreterDev` to add a bit specific about React & rethrow? Or does this sound too complicated?